drivers/interrupt_controller/intc_plic.c

```diff
@@ -98,15 +93,13 @@ int riscv_plic_irq_is_enabled(uint32_t irq)
 }
 
 /**
- *
  * @brief Set priority of a riscv PLIC-specific interrupt line
  *
  * This routine set the priority of a RISCV PLIC-specific interrupt line.
  * riscv_plic_irq_set_prio is called by riscv arch_irq_priority_set to set
  * the priority of an interrupt whenever CONFIG_RISCV_HAS_PLIC variable is set.
- * @param irq IRQ number for which to set priority
  *
- * @return N/A
+ * @param irq IRQ number for which to set priority
```

Comment:
Since you added a line for irq, can you add one for priority as well?

Reply:
Done.